src/main/java/de/tum/in/www1/artemis/web/rest/dto/PostContextFilter.java

```diff
@@ -8,115 +8,22 @@
 import de.tum.in.www1.artemis.domain.metis.PostSortCriterion;
 
 @JsonInclude(JsonInclude.Include.NON_EMPTY)
-public class PostContextFilter {
+public record PostContextFilter(@NotBlank Long courseId, long[] courseWideChannelIds, Long plagiarismCaseId, Long conversationId, String searchText, Boolean filterToUnresolved,
```

Comment:
Should we rename this class to `PostContextFilterDTO`? (And also its occurrences in variables to "...DTO")?